functions.inc.php

```diff
@@ -1323,9 +1327,10 @@ function _php_crypt_random_string($characters, $length)
  *
  * @param string $pw
  * @param string $pw_db optional encrypted password
+ * @param string $username optional
```

Comment:
can you add something to the username comment to say it's currently only relevant for dovecot DIGEST-MD5 format